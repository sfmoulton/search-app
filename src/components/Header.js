import React from 'react';
import {View, Text, StyleSheet} from 'react-native';

const Header = () => {
  return (
   <View style={styles.viewStyle}>
     <Text style={styles.textStyle}>Find My Breakfast</Text>
   </View>
  );
};

const styles = StyleSheet.create({
  viewStyle: {
    justifyContent: 'center',
    alignItems: 'center',
    height: 60,
    paddingTop: 15,
    shadowColor: '#000',
    shadowOffset: {width: 0, height: 2},
    shadowOpacity: 0.2
  },
  textStyle: {
    fontSize: 20
  }
})

export default Header;