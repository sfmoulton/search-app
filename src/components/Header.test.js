import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

describe('<Header />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree.children.length).toBe(1);
  });
  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
