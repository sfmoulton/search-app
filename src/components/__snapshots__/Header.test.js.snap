// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 60,
      "justifyContent": "center",
      "paddingTop": 15,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.2,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    Find My Breakfast
  </Text>
</View>
`;
